### DISCUSSION TITLE
How can I switch on/off fast charging via terminal?

### DISCUSSION THREAD

**Discussion by Chaos02** [VR-25/acc#145] [Ideas]
I am unsure if this is the right place to ask but I want to switch on/off (regular?) fast charging via terminal.
Since this is a repo, very familiar with handling charging on rooted android devices I thought I might get an answer here.
As far as I can tell, ACC does not have the ability to do this (yet?).
My intent is to only use fast charging if I really need it and not shred my battery otherwise. 
Greetings, Chaos!

**Comment by VR-25** [comment]
ACC does have this ability. The problem is that the control interface for charging speed varies across devices -- and some don't even have it. That said, a more compatible solution is limiting the charging current. Try `acc -sc 2000`, for 2000 mA.
